### PR TITLE
feat: Implement a bar chart to match latest design

### LINKED ISSUE #1102: Match bar chart implementation to match latest design (margin 2 px / 4 px)
**Description**

**Acceptance Criteria**
- [ ] match to final design on Figma (notice margins, etc..)

### PR DESCRIPTION
**Pull request by solid-tetianamovlian**
Fixes #1102 

## Checklist

- [x] Documented
- [x] Component tested

### CODE REVIEW

File: metrics/web/lib/common/presentation/metrics_theme/config/dimensions_config.dart [hunk 1]
```diff
@@ -13,11 +13,11 @@ class DimensionsConfig {
   static const double buttonHeight = 48.0;
 
   /// An outer diameter of the graph indicator.
-  static const double graphIndicatorOuterDiameter = 5.33;
+  static const double graphIndicatorOuterDiameter = 7.0;
```
Comment (solid-vladlaba) [inline metrics/web/lib/common/presentation/metrics_theme/config/dimensions_config.dart:16]: It seems like the indicator is `10.0` in diameter now: 
image

File: metrics/web/lib/common/presentation/metrics_theme/config/dimensions_config.dart [hunk 2]
```diff
@@ -13,11 +13,11 @@ class DimensionsConfig {
...
 
   /// An inner diameter of the graph indicator.
-  static const double graphIndicatorInnerDiameter = 2.13;
+  static const double graphIndicatorInnerDiameter = 2.8;
```
Comment (solid-vladlaba) [inline metrics/web/lib/common/presentation/metrics_theme/config/dimensions_config.dart:19]: It seems like the inner diameter is equal to `4.0`.
image

File: metrics/web/lib/dashboard/presentation/widgets/metrics_table_row.dart [hunk 3]
```diff
@@ -59,19 +59,19 @@ class MetricsTableRow extends StatelessWidget {
           status,
           Flexible(
             child: SizedBox(
-              width: 326.0,
+              width: 334.0,
```
Comment (solid-vladlaba) [inline metrics/web/lib/dashboard/presentation/widgets/metrics_table_row.dart:62]: Should we maybe rethink this moment as there is a `28.0` padding from the box with a name to the build results graph? Also, it appears that the whole tile has symmetric horizontal `32.0` padding, but now it is `const EdgeInsets.only(left: 32.0, right: 26.0)`.
image
image

**Review by solid-vladlaba — CHANGES_REQUESTED** [review]
Well done, @solid-maxim @solid-tetianamovlian! I've added a couple of suggestions below - please let me know what you think!

**Comment by solid-tetianamovlian** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by solid-vladlaba** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by solid-vladlaba — APPROVED** [review]
LGTM, thanks @solid-maxim @solid-tetianamovlian!:rocket: I've added minor fixes related to calculations and widget tree - please let me know what you think! 
Changes look good overall!

**Pull request merged**